session_0/word_processing.rb

```diff
@@ -34,11 +37,32 @@ def words_with_prefix(prefix, words)
 # The function `similarity_score` takes two words and returns the
 # similarity score (an integer).
 def similarity_score(word_1, word_2)
-  raise NotImplementedError # TODO
+  size = [word_1.size,word_2.size].min # TODO
+  score = 0
+  for i in 0..size-1
```

Comment:
What are you iterating over in the for loop?

Reply:
I was trying to extract each individual character in the word in the same way one does in C++. Is it possible to use word_1.each  to get individual characters?

Reply:
Experiment and find out